ci-operator/config/quay/quay-tests/quay-quay-tests-master__ocp-414-quay38.yaml

```diff
@@ -29,17 +29,28 @@ resources:
 tests:
 - as: quay-e2e-tests-quay38-ocp414-lp-interop
   cron: 0 6 * * 1
+  remote_api: true
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_TYPE: m5.2xlarge
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "OCPQE"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "OCPQE"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "OCPQE"},
+          {"step": "quay-tests-deploy-*", "failure_type": "all", "classification":  "Quay deployment", "jira_project": "OCPQE"},
+          {"step": "quay-tests-*", "failure_type": "test_failure", "classification":  "Test Failure", "jira_project": "OCPQE"},
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
     test:
     - ref: quay-tests-deploy-quay-aws-s3
     - ref: quay-tests-test-quay-e2e
     - ref: quay-tests-qbo-qe-test
     - ref: quay-tests-cso-qe-test
-    workflow: cucushift-installer-rehearse-aws-ipi
+    workflow: firewatch-ipi-aws
```

Comment:
@ascerra  the workflow "cucushift-installer-rehearse-aws-ipi" is defined by Openshift QE installation team, and mapping to profile "aws-qe", we should follow the rule. Otherwise we need to use other profile name to mapping your workflow.

Reply:
Sounds good I will update this thank you for the response.